turbopy/core.py

```diff
@@ -724,6 +724,89 @@ def interpval(yvec):
             return interpval
 
 
+    def set_cell_volumes(self, coordinate_system):
+        """
+        This function sets some volume-related elements that are useful for
```

Comment:
Save this docstring somewhere else and delete it from this file. It does not conform to the docstring standards we have set for this project.